LocationServices/LocationServices/Helpers/ErrorHandler.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  ErrorHandler.swift
+//  LocationServices
+//
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+
+import Foundation
+
+class ErrorHandler {
+    
+    static func isAWSStackDeletedError(error: any Error) -> Bool {
+        let errorMessage = error.localizedDescription.lowercased()
+        return errorMessage.contains("not found") || errorMessage.contains("security token included in the request is invalid")
```

Comment:
Have you checked that this message differs from token expiration error? So we won't mixup them